scalatest/src/main/scala/org/scalatest/tools/HtmlReporter.scala

```diff
@@ -781,7 +781,7 @@ private[scalatest] class HtmlReporter(
     
     def displayErrorMessage(errorMessage: String) = {
       // scala automatically change <br /> to <br></br>, which will cause 2 line breaks, use unparsedXml("<br />") to solve it.
-      val messageLines = errorMessage.split("\n")
+      val messageLines = errorMessage.split("\n").map(_.nn)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.